pkg/openshift/util/console_links.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"encoding/json"
+	"errors"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewLinkForApplicationMenu creates a ConsoleLink instance for the OpenShift console's Application menu
+func NewLinkForApplicationMenu(name string, host string, configMap *corev1.ConfigMap) (*consolev1.ConsoleLink, error) {
+	var consoleLink *consolev1.ConsoleLink
+
+	jsonStr, ok := configMap.Data["hawtconfig.json"]
```

Comment:
Ideally, we should declare a struct that represent the console config.

Reply:
Good point. I'll do that.

Reply:
I was more thinking about a struct defining the JSON structure, so that we could unmarshall the JSON into it and have it defining the API for the console configuration.

Reply:
I've defined a struct mapping to hawtconfig.json and unmarshalled the JSON into it. Is that what you had in mind?

Reply:
Exactly. Thanks!